deploy/eck-elasticsearch/examples/hot-warm-cold.yaml

```diff
@@ -0,0 +1,134 @@
+---
+nodeSets:
+- name: hot
+  count: 1
+  config:
+    node.roles: ["data_hot", "data_content", "ingest", "master"]
```

Comment:
If we are doing dedicated data tiers I think it would make sense to have dedicated masters as well